src/shadowbox/integration_test/test.sh

```diff
@@ -80,7 +80,6 @@ function cleanup() {
   status=$?
   if ((DEBUG != 1)); then
     docker-compose --project-name=integrationtest down
-    rm -rf ${TMP_STATE_DIR} || echo "Failed to cleanup files at ${TMP_STATE_DIR}"
```

Comment:
Why are you removing this?

Reply:
It's a source of flaky failures on both CI platforms -- sometimes we get a "No permission to remove".  It doesn't actually matter because the container gets torn down anyways, so there's no need to do cleanup ourselves.

Reply:
How is that flaky? It sounds like the problem may be something else? This has never been a problem before and it's good practice to clean up. I'd rather keep it there

Reply:
https://github.com/Jigsaw-Code/outline-server/runs/1841495356?check_suite_focus=true is an example.  It happens from time to time on Travis as well.

Reply:
I had never seen this, and this code has been there for years. Let's revert it for the purpose of this PR, and investigate if this happens again.

Perhaps the directories are being created as root and chown is not working somewhere else. I don't know, but maybe something else changed.

Reply:
Done in 360c6c683089766a5b758e3386671c5b064278b8